src/mixing.html

```diff
@@ -15,53 +15,78 @@ <h2 id="mixing">Annotating MathML</h2>
  and for MathML to embed markup from other vocabularies such as
  [[OpenMath]] or [[SVG]]. This general mechanism is described
  below in <a href="#mixing_semantic_annotations"></a> but first we describe a new
- feature added in MathML&#160;4 to faciliate annotating Presentation
- MathML with the <q>intent</q> or semantic meaning of the term.
+ feature added in MathML&#160;4 to facilitate annotating Presentation
+ MathML with the <q>intent</q> or (intended meaning) of the term.
  </p>
 
  <section>
   <h3 id="mixing_intent">The <code class="attribute">intent</code> attribute</h3>
-  <p>As decribed in <a href="#presm"></a> presentation MathML elements
+  <p>As decribed in <a href="#presm"></a> presentation <span class="ednote">Content?</span>
+  MathML elements
   (including [[MathML-Core]] allow attributes <code
   class="attribute">intent</code> and <code
   class="attribute">arg</code> that as described here allow the
-  <q>intent</q> or semantic meaning of the term to be specified. This
+  <dfn><q>intent</q></dfn> of the term to be specified. This
   annotation is not intented to provide a full mathematical definition
-  of the term but may be used guide Accessibility systems to give audio
+  of the term but may be used guide accessibility systems to give audio
   or braille renderings, see <a href="#accessibility"></a>. It may also
   be used to guide translations to Content MathML, or computational
   systems.</p>
 
+  <p>The <code class="attribute">intent</code> attribute encodes a
+  simple functional syntax representing  the intended meaning. A
+  formal grammar is given below, but a typical example would be
+  <code>intent="power($base,$exponent)"</code>. The parts consist:
+  of</p>
+  <ul>
+  <li><em>references</em> prefixed with <code>$</code>. A numeric
+  reference such as <code>$3/$1</code> references the [=intent=] of
+  the first child of the 3rd child of the element. An argumnt
+  reference such as <code>$name</code> refereces a descendent element
+  that has an attriute <code>arg="name"</code>. unlike <code class="attribute">id</code>
+  attributes, <code class="attribute">arg</code> do not have to be
+  unique within a document, when searching for a matching element the
+  search should only consider descendns and not consider descendents
+  of any element that has an <code class="attribute">intent</code> or
+  <code class="attribute">arg</code> attribute.</li>
+  <li><em>literal numbers</em> such as <code>2.5</code> denote
+  themselves. <span class="ednote">0.25e1?</span></li>
```

Comment:
Are you trying to get AT to pronounce "zero point twenty-five e 1"? What kind of `<mn>` (or other element) needs this type of annotation?

Reply:
It (again) depends what we want to say about defaults, if the default intent of an mn is its text content then `<mn>2.5</mn>` should (perhaps) be equivalent to `<mn intent="2.5">2.5</mn>` which means that should be allowed.

 Also if in general we want to allow intent to be "moved up the tree" to give less fine grained control but more control over how the entire expression is spoken then you need to allow

```
<mrow intent="times(2.5,$x)">
  <mn>2.5</mn>
   <mo>*</mo>
  <mi>x</mi>
</mrow>
```

Reply:
I'm not sure that's a *need*. The presentation tree is primary, and it can contain more information than the set allowed in the intent attribute itself. If we specify that

> the default behavior on leaf "token elements" is to read out their text content

then the English stating that suffices. 

If, instead, the aim was to allow rewriting arbitrary presentation MathML trees *into* intent values, then the "literal" grammar rule has to extend to all of Unicode, not just letters or numbers. Especially when we remember that `<mtext>` is also in the mix.